Clarify backup API (#3423)
mendix/docs: content/developerportal/operate/backups.md

See [Download a Backup](download-backup) for more information.

{{% alert type="info" %}}

> As the download archive is generated "on the fly" (meaning, while in progress), it is not possible to estimate the file size before downloading. Your browser will not show a progress bar.

Reviewer: This isn't 100% accurate actually. When we start creating the archive, we don't know how big it is. After it's been created, however, we do. (I don't think that we display it, however).

(The method of creating the archive is the same as when we use the API, so we request an archive to be created, and then when it's finally finished, we show the URL to the customer).

Reviewer 2: We do display the size of backup in CP  after its created. I'm fine with this rename.

Reviewer: My comment is not about the rename, it's about the overall statement.

I would suggest something like:
`As the download archive is generated on request, it's not possible to estimate the file size before requesting a download`

However, this isn't the most important thing.

Reviewer: I just checked, this isn't accurate, we do show a progress bar when downloading backups:

<img width="257" alt="image" src="https://user-images.githubusercontent.com/1497361/124570565-4ebd3880-de47-11eb-9175-99016a52a99d.png">

Author: ```suggestion
As the download archive is generated on request, it is not possible to estimate the file size before requesting a download.
```
I'll change it to this then Josh, @joshp123
